.github/ISSUE_TEMPLATE/Standard-Investigation-Issue-Template.md

```diff
@@ -0,0 +1,39 @@
+---
+name:  Investigation Issue
+about: Use this template for all Investigations
+labels: Investigation
+---
+# Story
+## Inspiration
+< What inspired this investigation >
+
+## Value of information acquired
+< What is the potential gain from the investigation outcome (customer dissatisfaction, loss of time, money, etc. >
+
+## Notes
+< special notes or suggestions information already acquired > 
+
+## Acceptance Criteria
+- [ ] A solution has been discovered and followup issue(s) / epic have been created
+- [ ] The proposed solution has been documented on this issue for approval
+
+or
+
+- [ ] A solution has been discovered but it was deemed not worth it
+
+## Subject Domain Experts
+< list of subject domain experts for this domain. If the subject expert from the end-user perspective is not the author, an end-user expert should be listed here. The Design, Dev, and QA teams should add experts here as appropriate. >
+
+## Progress
+### Reviewed Issues - completed by SM & Team Leads
+- [ ] Inspiration and Value complete
+- [ ] Initial Acceptance Criteria Defined
+- [ ] The person who will Approve story identified
```

Comment:
What about the "Subject Domain Experts" section? Even if it's not, someone should make it as so. I feel that not having a checkbox will leave open the opportunity for people to just ignore it.